stories/documentation/forms/timepicker/timepicker-basic.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { Meta, Story } from '@storybook/angular';
+
+interface timepickerBasicStory {
+	palette: string;
+	mod: string;
+	disabled: boolean;
+}
+
+export default {
+	title: 'Documentation/Forms/Timepicker/Basic',
+	argTypes: {
+		mod: {
+			options: ['', 'mod-outlined',],
+			control: {
+				type: 'radio',
+			},
+		},
+		palette: {
+			options: ['', 'palette-primary', 'palette-secondary', 'palette-grey', 'palette-success', 'palette-warning', 'palette-error'],
+			control: {
+				type: 'select',
+			},
+		},
+		disabled: {
+			control: {
+				type: 'boolean',
+			},
+		},
+	},
+} as Meta;
+
+function getTemplate(args: timepickerBasicStory): string {
+		const classes = [args.mod, args.palette].filter(Boolean).join(' ');
+		const disabled = args.disabled ? `disabled` : '';
+	return `
+<fieldset class="timepicker ${classes}" ${disabled}>
+	<legend class="u-mask">Durée</legend>
+	<div class="timepicker-field">
+		<span class="u-mask">heures</span>
+		<input class="timepicker-field-input" type="number" min="0" max="23" step="1" value="1" tabindex="1">
+		<button class="timepicker-field-increment" type="button" tabindex="-1" aria-hidden="true">
```

Comment:
Be careful if you put an implicit label, you will have to replace the buttons by links for the code to remain valid.